jui-core/index.d.ts

```diff
@@ -33,9 +33,9 @@ interface UtilBase {
      * @param {Function} ctor base Class
      * @param {Function} superCtor super Class
      */
-    // tslint:disable-next-line:callable-types
+    /* tslint:disable-next-line:callable-types */
```

Comment:
This isn't accomplishing anything. The lint error is that you have `inherit (` instead of `inherit(`.
You can run `npm run lint jui-core` locally instead of waiting for travis to lint.